examples/llama2/llama2_qlora.json

```diff
@@ -0,0 +1,135 @@
+{
+    "input_model":{
+        "type": "PyTorchModel",
+        "config": {
+            "model_script": "user_script.py",
+            "io_config": "get_merged_decoder_with_past_io_config",
```

Comment:
Nit: the io_config name is a little bit confused since it is a function. By contrast, the next line dummpy_inputs_func has a suffix func.

Are there any best practice to normalize these naming?

Reply:
I think what happened is that some earlier PR made io_config: dict, str, func when it was originally only a dict. 
@trajepl has created a related TODO https://github.com/microsoft/Olive/blob/main/olive/model/hf_utils.py#L25

I can create a new work item to track this.